### PR TITLE
Create pdf package

### PR DESCRIPTION
**Pull request by rkondratowicz**
cleanup after https://github.com/hmcts/send-letter-service/pull/107

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java [hunk 1]
```diff
@@ -17,7 +17,8 @@
 import uk.gov.hmcts.reform.sendletter.entity.Letter;
 import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
 import uk.gov.hmcts.reform.sendletter.model.in.LetterRequest;
-import uk.gov.hmcts.reform.sendletter.services.util.DuplexPreparator;
+import uk.gov.hmcts.reform.sendletter.services.pdf.DuplexPreparator;
+import uk.gov.hmcts.reform.sendletter.services.pdf.PdfCreator;
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java]: I think this should not be part of this PR.This is also imported in other files below

Comment (rkondratowicz) [reply src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java]: package changed, it is needed now
